wgpu/tests/partially_bounded_arrays/mod.rs

```diff
@@ -0,0 +1,156 @@
+use std::{borrow::Cow, num::NonZeroU32};
+
+use crate::common::{initialize_test, TestParameters};
+use wasm_bindgen_test::*;
+
+#[test]
+#[wasm_bindgen_test]
+fn partially_bounded_array() {
+    initialize_test(
+        TestParameters::default()
+            .features(
+                wgpu::Features::TEXTURE_BINDING_ARRAY
+                    | wgpu::Features::STORAGE_RESOURCE_BINDING_ARRAY
+                    | wgpu::Features::PARTIALLY_BOUND_BINDING_ARRAY
+                    | wgpu::Features::TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES,
+            )
+            .limits(wgpu::Limits {
+                ..wgpu::Limits::downlevel_defaults()
+            })
+            .backend_failure(
+                wgpu::Backends::GL
+                    | wgpu::Backends::DX11
+                    | wgpu::Backends::METAL
+                    | wgpu::Backends::DX12
+                    | wgpu::Backends::BROWSER_WEBGPU,
+            ),
```

Comment:
This shouldn't be necessary - those backends will all be skipped as they don't have the required features